.github/workflows/weblate-sync-po.yml

```diff
@@ -0,0 +1,46 @@
+name: weblate-sync-po
+on:
+  schedule:
+    # Run this on Tuesday evening (UTC), so that it's ready for release on
+    # Wednesday, with some spare time
+    - cron: '0 18 * * 2'
+  # can be run manually on https://github.com/cockpit-project/cockpit/actions
```

Comment:
This is cockpit-files

Reply:
This is copy-pasta from machines, which apparently has the same problem :)